README.md

```diff
@@ -225,7 +172,7 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
+- [ ] **[add reducers and actions]**
 - [ ] **[new_feature_2]**
 - [ ] **[new_feature_3]**
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea we remove the placeholder text under the _Future Features_ section. We can add them anytime we have new features in mind!